### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
In this PR:
- created table vets, establishing many-to-many relationships.
- created join tables (specialization, visits) to handle relationships.
- made queries using join tables.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -30,8 +30,32 @@ ALTER TABLE animals
...
+ALTER TABLE animals ADD COLUMN owner_id INTEGER;
+ALTER TABLE animals 
    ADD CONSTRAINT fk_owners 
    FOREIGN KEY(owner_id)
-   REFERENCES owners(id);
\ No newline at end of file
+   REFERENCES owners(id);
+
+CREATE TABLE vets (
+   id serial PRIMARY KEY NOT NULL,
```
Comment (arslanbisharat) [inline schema.sql:40]: - [ ] Please make sure 'id' is an integer.

File: data.sql [hunk 2]
```diff
@@ -48,4 +48,37 @@ update animals set owner_id = 5 where name like 'Charmander';
...
 update animals set owner_id = 5 where name like 'Blossom';
 update animals set owner_id = 6 where name like 'Angemon';
-update animals set owner_id = 6 where name like 'Boarmon';
\ No newline at end of file
+update animals set owner_id = 6 where name like 'Boarmon';
+
+insert into vets (name, age, date_of_graduation) values
+  ('William Tatcher', 45, '2000-04-23'), 
+  ('Maisy Smith', 26, '2019-01-17'), 
+  ('Stephanie Mendez', 64, '1981-05-04'),
```
Comment (arslanbisharat) [inline data.sql:56]: - [ ] Please update the date for `Stephanie Mendez` for `1981-05-17`.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Arturgouveia1970, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Arturgouveia1970) [reply schema.sql:40]: hello @arslanbisharat , acording to all documentation that I have seen, when use SERIAL, does that. this I copy from geeksforgeeks: 
By assigning the SERIAL to the id column, PostgreSQL carries out the following: 

The initial step generates sequences of integers, where the next default value depends upon the value for the column.
As SERIAL always generates a sequence of integers, it is important to set that no null value is added as an ID to any column. This is done by adding the NOT NULL constraint.
Finally, it is needed to set the owner of the sequence for the ID column. These IDs are automatically deleted if the column or table is dropped.

File: data.sql (hunk 2 above)
Comment (Arturgouveia1970) [reply data.sql:56]: hello @arslanbisharat ,I do not understand this required change either. acording to the exercise requirements: 
Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
can you please advise me on how to procced
Artur

Comment (rotimiazeez) [reply data.sql:56]: - I think this is a mix up between the project requirement and code review guideline, but you are good to go on this.👏

**Review by rotimiazeez — APPROVED** [review]
# Hi @Arturgouveia1970 👍🏾, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**